cmd/agent/subcommands/analyzelogs/command.go

```diff
@@ -63,22 +76,34 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 			return fxutil.OneShot(runAnalyzeLogs,
 				core.Bundle(),
 				fx.Supply(cliParams),
-				fx.Supply(command.GetDefaultCoreBundleParams(cliParams.GlobalParams)),
+				fx.Supply(core.BundleParams{
+					ConfigParams: config.NewAgentParams(globalParams.ConfFilePath, config.WithFleetPoliciesDirPath(globalParams.FleetPoliciesDirPath)),
+					SecretParams: secrets.NewEnabledParams(),
+					LogParams:    log.ForOneShot("", "off", true)}),
+				dualTaggerfx.Module(common.DualTaggerParams()),
+				workloadmetafx.Module(defaults.DefaultParams()),
+				autodiscoveryimpl.Module(),
 			)
 		},
 	}
 
 	// Add flag for core config (optional)
 	cmd.Flags().StringVarP(&cliParams.CoreConfigPath, "core-config", "C", defaultCoreConfigPath, "Path to the core configuration file (optional)")
+	// Add flag for inactivity timeout (optional)
+	cmd.Flags().IntVarP(&cliParams.inactivityTimeout, "inactivity-timeout", "t", 1, "Time (seconds) that the program will wait for new logs before exiting (optional)")
```

Comment:
Instead of an `Int` - we can use a duration. This allows users to specify the interval ex:
`1s` or `10m`

[Here is an example](https://github.com/DataDog/datadog-agent/blob/b06928ce6589f5a371b8bf5e96e023638b7b0cad/cmd/agent/subcommands/dogstatsdcapture/command.go#L67)